mirrord/operator/src/crd.rs

```diff
@@ -144,3 +144,23 @@ pub struct CopyTargetSpec {
     /// Ignored if [`Target`] is not [`Target::Deployment`].
     pub scale_down: bool,
 }
+
+/// Custom resource for policies that limit what mirrord features users can use.
+#[derive(CustomResource, Clone, Debug, Deserialize, Serialize, JsonSchema)]
+#[kube(
+    group = "operator.metalbear.co",
+    version = "v1",
+    kind = "MirrordPolicy",
+    root = "MirrordPolicyCrd",
+    namespaced
+)]
+pub struct MirrordPolicySpec {
+    /// Specify the targets for which this policy applies, in the pod/my-pod deploy/my-deploy
+    /// notation. Targets can be matched using `*` and `?` where `?` matches exactly one
+    /// occurrence of any character and `*` matches arbitrary many (including zero) occurrences
+    /// of any character. If `None`, the policy will apply for all targets.
+    pub target: Option<String>,
```

Comment:
it'd be better if we do it based on matching labels, but if it's a lot more work we can have it next version.

Reply:
Shouldn't be that hard, but I think @eyalb181 didn't want a label selector there.

Reply:
@aviramha Wouldn't label selector require the customer to add labels to any deployments/pods they want to restrict?

Reply:
Usually applications have an application label, or name so they can just use that, and it'd make more sense to have a label
`mirrord: nostealpolicy` to make pods like that.
I'm up for both options, just need to leave room (for example making not specifying target that it'd apply to all would be a lie if you'd then add labels)

Reply:
Question about selectors:
On deployment runs - should we check the individual pods' labels or just the deployment's?